business_requirement/models/business.py

```diff
@@ -202,6 +204,208 @@ def action_button_cancel(self):
     def action_button_drop(self):
         self.write({'state': 'drop'})
 
+    @api.cr_uid_ids_context
+    def message_post(self, cr, uid, thread_id, body='', subject=None,
+                     type='notification', subtype=None, parent_id=False,
+                     attachments=None, context=None,
+                     content_subtype='html', **kwargs):
+        # ---------------------------- changes start here 2016/03/08
+        """
+        Overwrite method message_post from mail.thread to modify the default
+        behavior of subject with mail messages.
+        """
+        # ---------------------------- changes start here 2016/03/08
+        """ Post a new message in an existing thread, returning the new
+            mail.message ID.
+
+            :param int thread_id: thread ID to post into, or list with one ID;
+                if False/0, mail.message model will also be set as False
+            :param str body: body of the message, usually raw HTML that will
+                be sanitized
+            :param str type: see mail_message.type field
+            :param str content_subtype:: if plaintext: convert body into html
+            :param int parent_id: handle reply to a previous message by adding
+                the parent partners to the message in case of private
+                discussion
+            :param tuple(str,str) attachments or list id: list of attachment
+                tuples in the form ``(name,content)``, where content is NOT
+                base64 encoded
+
+            Extra keyword arguments will be used as default column values
+            for the new mail.message record. Special cases:
+                - attachment_ids: supposed not attached to any document;
+                    attach them to the related document.
+                    Should only be set by Chatter.
+            :return int: ID of newly created mail.message
+        """
+        # ---------------------------- changes start here 2016/03/08
```

Comment:
@siyuancn, @duanyp1991 give the order to add this comment, because it's easy to follow the change between the original. 

cc @duanyp1991